### PR TITLE
Update QUEST_LV_0200.tsv

### PR DESCRIPTION
**Pull request by Requinn**
Checked through lines 000001 - 002470:
Translated lines 005368, 005381, 006556 - 006591, and 007061 - 007065.

Various grammatical fixes.
Translated a few blank lines.
Titles fixed. (i.e. the guard, X >>> Guard X)
Names fixed to match their English counterparts. (i.e. Yohan >>> John)
- Rofdel on line 002361 eludes me though.
- Line 002378: Wilhelmina is a real name. (was a queen of the
  netherlands) Not sure what Carriot could    possibly be if its not a
  last name.

(Theres so much I want to fix aaaaa-)

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -198,13 +198,13 @@ QUEST_LV_0200_20150317_000197	Because of our plan, we may be sent to the gallows
...
 QUEST_LV_0200_20150317_000202	Because of the petrified frosts that were created due to Ruklys' rebellion a few hundreds years ago, the land turned into a wasteland.
 xQUEST_LV_0200_20150317_000203	We will allow you to enter since Kalis Knightage requested us to do so, but please stay where our gaze remains.
 QUEST_LV_0200_20150317_000204	After the Medis Diena 4 years ago, Benas was appointed as the chief commander for the forces here.{nl}He ordered to take good care of the road leading to the Fortress of the Earth.
 QUEST_LV_0200_20150317_000205	If you are not in a hurry, please defeat the monsters in area A.{nl}That will be the first step for you to contribute to the Goddess and the Kingdom..{nl}Of course, I will promise you enough rewards.
-QUEST_LV_0200_20150317_000206	The commander of the rebel force, Ruklys, and the citizens here engaged in an evil conduct{nl}despite the selections from the great merchant, Premier Eminent. The result is this place.
-QUEST_LV_0200_20150317_000207	
+QUEST_LV_0200_20150317_000206   Rebel Commaner Ruklys and the citizens here engaged in an evil conduct{nl}despite the selections from the great merchant, Premier Eminent. The result is this place.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Extra space?

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -384,16 +384,16 @@ QUEST_LV_0200_20150317_000383	There is a monocle that helps you identify preciou
 QUEST_LV_0200_20150317_000384	The scroll Bilhelmina gave us is awesome!{nl}I didn't hear a thing even from right beside.
 QUEST_LV_0200_20150317_000385	Shall we go now?
 QUEST_LV_0200_20150317_000386	I'm a wizard who can't use any magic. I feel pathetic.
-QUEST_LV_0200_20150317_000387	Antares.. Used to be a great wizard back in the day.
+QUEST_LV_0200_20150317_000387	Antares used to be a great wizard back in the day.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: I think the previous line is trying to create a pause after Antares. Try adding an ellipse … so we maintain that. :)

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -402,11 +402,11 @@ xQUEST_LV_0200_20150317_000401	Recently, monsters have been swarming out of the
 xQUEST_LV_0200_20150317_000402	After much debate on how to solve the situation, your name was brought up. {nl}That you will gladly come forward to hunt down the monsters. {nl}
 xQUEST_LV_0200_20150317_000403	I don't know how to thank you for accepting such a difficult mission.. {nl}Well then, please take care of the monsters in Purple Tree Fault Forest. {nl}Purple Tree Fault Forest is located past Fedimian, on the way to the Cathedral.
 xQUEST_LV_0200_20150317_000404	Good job. {nl}I did hear a lot about how great you were, but it seems like it wasn't all just rumours.
-xQUEST_LV_0200_20150317_000405	Huh, tsk tsk.. What a pity. {nl}Of course, a merchant like me won't fully understand it.. {nl}
+xQUEST_LV_0200_20150317_000405	Huh, tsk tsk... What a pity. {nl}Of course, a merchant like me won't fully understand it.. {nl}
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Change to ellipse … character.

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -402,11 +402,11 @@ xQUEST_LV_0200_20150317_000401	Recently, monsters have been swarming out of the
...
 QUEST_LV_0200_20150317_000406	But if you help me out, I will give you something. It may not be as great as what you broke but it's pretty good.
 xQUEST_LV_0200_20150317_000407	It's nothing much, really. {nl}The symbol of pilgrim on the Starving Demon's Road outside Fedimian will do.
 xQUEST_LV_0200_20150317_000408	Nevermind if you can't. {nl}But if you can, that'd be awesome.
-xQUEST_LV_0200_20150317_000409	Oh, you got it.. {nl}Well, aren't reckless challenges the fun in life?
+xQUEST_LV_0200_20150317_000409	Oh, you got it... {nl}Well, aren't reckless challenges the fun in life?
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Change to ellipse … character.

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -452,8 +452,8 @@ xQUEST_LV_0200_20150317_000451	Oh, you got them. Thank you. {nl}Please go back t
 QUEST_LV_0200_20150317_000452	Terrible things are happening. {nl}Can we go back to the peaceful days?
 xQUEST_LV_0200_20150317_000453	The Goddess can be so cruel. {nl}Please pray that my colleagues may rest in peace.
 QUEST_LV_0200_20150317_000454	Pilgrim Lylia
-QUEST_LV_0200_20150317_000455	If I die, please bury me..{nl}Make my body unseen..{nl}If I lose my breath, please bury me..
-xQUEST_LV_0200_20150317_000456	And if you meet a pilgrim named Julius, please tell him these words.. tell him goodbye..
+QUEST_LV_0200_20150317_000455	If I die, please bury me...{nl}Make my body unseen...{nl}If I lose my breath, please bury me...
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Change to ellipse … character.

File: QUEST_LV_0200.tsv [hunk 6]
```diff
@@ -452,8 +452,8 @@ xQUEST_LV_0200_20150317_000451	Oh, you got them. Thank you. {nl}Please go back t
...
+xQUEST_LV_0200_20150317_000456	And if you meet a pilgrim named Julius, please tell him these words... tell him goodbye...
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Change to ellipse … character.

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -1003,19 +1003,19 @@ QUEST_LV_0200_20150317_001002	I followed because I was worried about Darren but
...
 QUEST_LV_0200_20150317_001007	He started getting mad at me since some time ago and now he wouldn't even look at me. {nl}I have no clue why. I should know the reason at least to apologize or something.
 QUEST_LV_0200_20150317_001008	Oh, that is some sweet advice. {nl}He wouldn't get mad at someone he sees for the first time, would he?
 QUEST_LV_0200_20150317_001009	Why do you think he is mad? {nl}Can you guess why?
 QUEST_LV_0200_20150317_001010	Mikolas
 QUEST_LV_0200_20150317_001011	Oh Cleopas.. What a shame on him to ask someone else's help. {nl}He borrowed my brewing machine, sold it somewhere, and now acts like he doesn't know about it?
 QUEST_LV_0200_20150317_001012	How could he do that to me when I was so good to him? {nl}He even lied about it. He said he placed it in front of my workshop but I never received it.
-QUEST_LV_0200_20150317_001013	Is Micolas not ready yet?
-QUEST_LV_0200_20150317_001014	How could he do this to me?{nl}Micolas..
+QUEST_LV_0200_20150317_001013	Is Mikolas not ready yet?
+QUEST_LV_0200_20150317_001014	How could he do this to me?{nl}Mikolas..
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv:1014]: Add ellipse … character.

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -836,7 +836,7 @@ QUEST_LV_0200_20150317_000835	When will this pitiful feeling end. {nl}When I loo
 xQUEST_LV_0200_20150317_000836	I can't help that the supplies or backup is delayed. {nl}But tellin us to just wait is too much.
 QUEST_LV_0200_20150317_000837	I'm afraid where the monsters will jump out from. {nl}I plan to run away if an unknown monster shows up.
 QUEST_LV_0200_20150317_000838	We're making up for the lacking supplies. {nl}If the supplies don't arrive, we have to cover for it on site.
-QUEST_LV_0200_20150317_000839	Somehow, at least sir Philipas' troops..
+QUEST_LV_0200_20150317_000839	Somehow, at least sir Philip's troops..
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Add ellipse … character.

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -1023,9 +1023,9 @@ xQUEST_LV_0200_20150317_001022	Us?{nl}We are best friends who grew up in the sam
 QUEST_LV_0200_20150317_001023	By the way, why are you asking?
 QUEST_LV_0200_20150317_001024	Ahah, thank you so much.{nl}You will be able to obtain royal jellies form the monsters nearby.
 xQUEST_LV_0200_20150317_001025	If it weren't you, we would have never gotten closer again.
-QUEST_LV_0200_20150317_001026	Um? Isn't this royal jellly?{nl}Micolas.. It is so fortunate. I was able to get rid of the misunderstanding because of you.{nl}May the Austeya bless you.
+QUEST_LV_0200_20150317_001026	Um? Isn't this royal jellly?{nl}Mikolas.. It is so fortunate. I was able to get rid of the misunderstanding because of you.{nl}May Austeya bless you.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Jelly
Add  ellipse … character.

File: QUEST_LV_0200.tsv [hunk 10]
```diff
@@ -2155,112 +2155,112 @@ xQUEST_LV_0200_20150317_002154	The supply officer told you that the soliders are
...
+QUEST_LV_0200_20150317_002217	Talk to Kingdom Army Retia
+QUEST_LV_0200_20150317_002218	Talk to Kingdom Army Retia
+QUEST_LV_0200_20150317_002219	Kingdom Army Retia is waiting for someone's help at the Square of Lamentation
+QUEST_LV_0200_20150317_002220	Kingdom Army Retia asked you to defeat the monsters at A region.
+QUEST_LV_0200_20150317_002221	You defeated the monsters that Kingdom Army Retia requested from you. Go back to Retia.
 QUEST_LV_0200_20150317_002222	Check the petrification device
-QUEST_LV_0200_20150317_002223	The Kingdom Army, Retia has asked you to check the petrification devices that are installed at various places in the Square of Lamentation.
-QUEST_LV_0200_20150317_002224	As the Kingdom Army, Retia requested to you, you checked the petrificaion devices. The reactions on the devices are abnormal. Let's go back to Retia.
+QUEST_LV_0200_20150317_002223	Kingdom Army Retia has asked you to check the petrification devices that are installed at various places in the Square of Lamentation.
+QUEST_LV_0200_20150317_002224   You check the petrification devices as Kingdom Army Retia had requested. The reactions on the devices are abnormal. Let's go back to Retia.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: Extra space

File: QUEST_LV_0200.tsv [hunk 11]
```diff
@@ -2155,112 +2155,112 @@ xQUEST_LV_0200_20150317_002154	The supply officer told you that the soliders are
...
 QUEST_LV_0200_20150317_002256	Defeat Saltistter
-QUEST_LV_0200_20150317_002257	Talk to the guard, Gofden
+QUEST_LV_0200_20150317_002257	Talk to Guard Gofden
 xQUEST_LV_0200_20150317_002258	Guard Gofden is waiting for someone's help at the gloomy market place in the petrified city.
 xQUEST_LV_0200_20150317_002259	Guard Gofden asked you to help here. Defeat the monsters nearby to help Gofden.
 xQUEST_LV_0200_20150317_002260	You've defeated all the monsters nearby as the guard Gofden requested of you. Let's go back to Gofden.
 QUEST_LV_0200_20150317_002261	Destroy the records of the rebels
 xQUEST_LV_0200_20150317_002262	Guard Gofden asked you to destroy the records of the rebles that were left neglected for a long time. You can find the records of the rebels from the commerce district.
-xQUEST_LV_0200_20150317_002263	As the guard Gofden requested of you, you've destroyed most of the rebel's records. Let's go back to Gofden.
+xQUEST_LV_0200_20150317_002263  You've destoyred the thieve's recorde like Guard Gofden asked. Let's go back to Gofden.
```
Comment (ttgmichael) [inline QUEST_LV_0200.tsv]: extra space?

**Comment by Requinn** [comment]
ok, so we are changing all of them to three periods? (instead of spamming this page, can just ctrl-F them in later)

**Comment by ttgmichael** [comment]
Yep! IMC ok'd the use of the ellipse character. I think it makes the text more shorter too.

Sorry for spamming xD. (Thought I'd cover every line ^^)

Thanks, a lot of your edits are great! :)

**Comment by ttgmichael** [comment]
Oh, and next time, it'll be more preferable to make smaller pull requests. you almost hit the 300 line limit. We made some guidelines that we wish more people could follow to keep this project organized and bug free :) https://github.com/Treeofsavior/EnglishTranslation/issues/540

**Comment by Requinn** [comment]
Alright, thanks for the heads up on that. And sorry, I knew about the limit but I forgot I had to put back edits from that reversion earlier :T

**Comment by ghost** [comment]
Again I want to stress that we shouldn't use three dot ellipses in places where only two are used in the original Korean text.

Alright I'll stop nitpicking now. :>

**Comment by Ihysoal** [comment]
An ellipse is a circle. An ellipsis however, is a marker used for a variety of things, commonly used by using 3 dots in succession like you are!

> jalevijo commented 5 minutes ago
> Again I want to stress that we shouldn't use three dot ellipses in places where only two are used in the original Korean text.

Did imcgames state this somewhere? You should probably link to that suggestion or whatnot then. 2 dot ellipsis aren't a thing.

Furthermore, ttgmichael stated that imcgames already gave the ok on using the specific ellipsis character, so who's right here?

**Comment by ghost** [comment]
http://puu.sh/jnC27.png

It wasn't stated anywhere but the files themselves. Some (most) are "..", and others are actual ellipses. While there's no such thing as a 2-dot ellipsis, two periods are still used to convey a longer pause than a single period, and making that pause even longer in sentences where it doesn't need to be makes them sound awkward.

Anyway, it's nitpicking like I said. I won't complain about it anymore, but I do want to make sure my point was understood. I just wanted the punctuation to stay consistent.

**Comment by Requinn** [comment]
Perhaps something to begin an issues thread about for imcgames to see?

**Comment by ghost** [comment]
No let's not give them a headache over some dots heheh. If they're fine with your edits then leave them as they are.

**Comment by Ihysoal** [comment]
I think (and hope they're working on it) that imcgames needs to get it together and release some actual writing standards. Either hire a professional writer or work together with the high-quality contributors like ensata.

There are too many cooks in the kitchen. I don't even want to attempt to contribute because of all the conflicts.

**Comment by ghost** [comment]
Well professional writers and translators are expensive, which is why we're all here together! Working more closely with individuals and actually planning could potentially be more work for IMC than simply denying the bad pull requests and accepting the good ones. I'm not actually sure.

**Comment by Requinn** [comment]
Should I update this again to the latest file version? I know they're not taking any new PRs during the beta period, but it does have all of the missing translations until line 007065 filled in (except for 4 lines). There's also a new PR with lines that could conflict with this one. Just want to know so I can start translating the rest of the file.

**Pull request closed without merging**